pronto_mia/lib/ui/views/deployment_plan/edit/deployment_plan_edit_viewmodel.dart

```diff
@@ -1,31 +1,40 @@
-import 'package:file_picker/file_picker.dart';
+import 'package:flutter/foundation.dart';
 import 'package:stacked/stacked.dart';
 import 'package:stacked_services/stacked_services.dart';
 
 import 'package:pronto_mia/app/app.router.dart';
 import 'package:pronto_mia/app/service_locator.dart';
 import 'package:pronto_mia/core/services/deployment_plan_service.dart';
 import 'package:pronto_mia/ui/views/deployment_plan/edit/deployment_plan_edit_view.form.dart';
+import 'package:pronto_mia/core/models/file_upload.dart';
+import 'package:pronto_mia/core/factories/error_message_factory.dart';
+import 'package:pronto_mia/core/models/deployment_plan.dart';
 
 class DeploymentPlanEditViewModel extends FormViewModel {
   DeploymentPlanService get _deploymentPlanService =>
       locator.get<DeploymentPlanService>();
   NavigationService get _navigationService => locator.get<NavigationService>();
+  ErrorMessageFactory get _errorMessageFactory =>

```

Comment:
Why do we have to get this class over the locator in the first place? Isn't it a static helper class? So we could make getErrorMessage static?

Reply:
As discussed we can implement the method static, but this will be done in a later pull request.